src/main/java/com/spotify/sparkey/Util.java

```diff
@@ -266,41 +267,46 @@ static void nonThrowingClose(Closeable stream) {
     }
   }
 
-  static boolean renameFile(File srcFile, File destFile) throws FileNotFoundException {
+  static void renameFile(File srcFile, File destFile) throws IOException {
```

Comment:
This method could really benefit from using the NIO File API instead (which has been around since Java 7 anyway)

Reply:
That might be a good idea - saving it for later though.